### ISSUE TITLE
what is the metrics of  BlendedLoss()?

### ISSUE THREAD

**Issue by Ahandsomenaive** [leeesangwon/PyTorch-Image-Retrieval#14]
Thank you that you can share your codes with others. 
then i do not understand a question in codes.
class BlendedLoss(object):
    def __init__(self, main_loss_type, cross_entropy_flag):
        super(BlendedLoss, self).__init__()
        self.main_loss_type = main_loss_type
        assert main_loss_type in MAIN_LOSS_CHOICES, "invalid main loss: %s" % main_loss_type

        self.metrics = []

metrics is one set of some metric functions??
here you are not introduction it?
hope your reply! thank you

**Comment by leeesangwon** [comment]
Before we make the codes public, there was online-triplet loss implementation which is from [here](https://github.com/adambielski/siamese-triplet).
Because `self.metrics` is only for the online-triplet loss, there is no meaning for now.
I'll remove that part soon.
Thank you.

**Comment by Ahandsomenaive** [comment]
> Before we make the codes public, there was online-triplet loss implementation which is from [here](https://github.com/adambielski/siamese-triplet).
> Because `self.metrics` is only for the online-triplet loss, there is no meaning for now.
> I'll remove that part soon.
> Thank you.

Thank you.
